src/sensitivities/chainrules.jl

```diff
@@ -176,16 +188,26 @@ function overload_declarations!(signature_def)
         primal_val, pullback = rrule(op, unbox.(args)...; kwargs...)
         tape = get_tape(args)
 
-        branch = Branch(primal_val, op, args, kwargs.data, tape, length(tape) + 1, pullback)
+        branch = Branch(
+            primal_val, op, args, values(kwargs), tape, length(tape) + 1, pullback
```

Comment:
FWIW, isn't the `values` change unrelated to this PR?

Reply:
Correct unrelated to this PR.